controllers/namespace/namespace_controller.go

```diff
@@ -23,6 +23,9 @@ import (
 // once is a pointer to the sync.Once type
 var once sync.Once
 
+// performCSIAddonsPodRestart a bool type to check if the csi-Addons pod should be restarted
+var performCSIAddonsPodRestart bool
```

Comment:
as its global should we take lock. not required now but if we enable max-concurrent-reconciles we will get into problems

Reply:
I think we are fine for now, we can update it later on when we enable max-concurrent-reconciles.